crates/nargo/src/package.rs

```diff
@@ -30,4 +59,12 @@ impl Package {
         // For now it is hard-coded to be toml.
         self.root_dir.join(format!("{VERIFIER_INPUT_FILE}.toml"))
     }
+
+    pub fn is_binary(&self) -> bool {
+        matches!(self.package_type, PackageType::Binary)
+    }
+
+    pub fn is_library(&self) -> bool {
+        matches!(self.package_type, PackageType::Library)
+    }
```

Comment:
We should probably implement `Eq` on `PackageType` and turn these into `self.package_type == PackageType::Binary`, etc.

Reply:
+1

Reply:
Done, but I don't really like it.